py/fetcher/main.py

```diff
@@ -0,0 +1,160 @@
+import certifi
+import logging
+
+from datadog_api_client import ApiClient, Configuration
+from datadog_api_client.v1.api.metrics_api import MetricsApi
```

Comment:
This means you would have to import datadog. 
Which means everybody using this API will need to import Datadog. Unless this is actually needed, which does not seem the case as you are using only one method which is quite simple, consider sending a bare HTTP request with urllib 

It is documented here https://docs.datadoghq.com/api/latest/metrics/?code-lang=curl#query-timeseries-points
(the curl section tells you how to craft the request).

The alternative is that you package your metrics cronjob in a separate package from the usage_accountant library so you have separate lists of dependencies. But that will take you longer to refactor the project structure.